src/model/game/GameManager.ts

```diff
@@ -562,6 +562,19 @@ export default class GameManager {
             "https://thunderstore.io/c/sailwind/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
             [new StorePlatformMetadata(StorePlatform.STEAM, "1764530")], "Sailwind.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, []),
+        new Game(
+            "Voices of the Void", "VotV", "VotV",
+            "", ["VotV-Win64-Shipping.exe"], "",
```

Comment:
Should the data folder name for VotV be defined? the mod linker now depends on it for ue4ss.dll & dwmapi.dll placement logic, so just checking if this is actually correct? (it's the empty string at the end atm)

Reply:
Yeah, that should be set to `VotV`. Will fix.